sycl/test-e2e/format.py

```diff
@@ -334,6 +334,12 @@ def get_extra_env(sycl_devices):
                 expanded = "env"
 
                 extra_env = get_extra_env([parsed_dev_name])
+                backend, device = parsed_dev_name.split(":", 1)
```

Comment:
The example of the intended lit string after the change in the PR description is 

```
llvm-lit --param "sycl_devices=level_zero_v2:arch-intel_gpu_mtl_u" sycl/test-e2e
```

but I was expecting something like

```
llvm-lit --param "sycl_devices='level_zero_v1:gpu;level_zero_v2:gpu" sycl/test-e2e
```

Can you clarify on the intended use case? Thanks